data/index.js

```diff
@@ -79,6 +66,34 @@ export function registerSelectors( reducerKey, newSelectors ) {
 	selectors[ reducerKey ] = mapValues( newSelectors, createStateSelector );
 }
 
+/**
+ * Registers actions for external usage.
+ *
+ * @param {string} reducerKey   Part of the state shape to register the
+ *                              selectors for.
+ * @param {Object} newActions   Actions to register.
+ */
+export function registerActions( reducerKey, newActions ) {
+	const store = stores[ reducerKey ];
+	const createBoundAction = ( action ) => ( ...args ) => store.dispatch( action( ...args ) );
+	actions[ reducerKey ] = mapValues( newActions, createBoundAction );
```

Comment:
At the moment every `registerActions` call overwrites stored action for the given reducer key. Should we prevent calling this method more than once or allow to merge actions? The same applies to `registerSelectors` and `registerReducer`.

Reply:
As I said above, we could consolidate all these three function with a single `registerState`